admin/config-ui/components/class-component-connect-google-search-console.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * @package WPSEO\Admin\ConfigurationUI
+ */
+
+/**
+ * Class WPSEO_Config_Component_Connect_Google_Search_Console
+ */
+class WPSEO_Config_Component_Connect_Google_Search_Console implements WPSEO_Config_Component {
+
+	const OPTION_ACCESS_TOKEN = 'wpseo-gsc-access_token';
```

Comment:
They are currently implied by providing a "prefix" dynamically.
The GSC component is a dependency in Yoast SEO.